# scRNAseq-pipeline

## Snakemake to generate automated workflows

+ Read [snakemake tutorial](https://snakemake.readthedocs.io/en/stable/).
+ Example of building a snakefile for [sequencing analysis](http://pedagogix-tagc.univ-mrs.fr/courses/ABD/practical/snakemake/snake_intro.html).
+ Github [beginners snakemake tutorial](https://github.com/leipzig/SandwichesWithSnakemake)
+ Github tutorial on [seq analysis](https://github.com/leipzig/snakemake-example/blob/master/Snakefile)
+ [ChIP-seq snakemake pipeline](https://molb7621.github.io/workshop/Classes/snakemake-tutorial.html)

## Pseudoalignment integration

You can run [kallisto](http://pachterlab.github.io/kallisto/) within the scater package.
Click [here](https://rdrr.io/bioc/scater/man/runKallisto.html) for an example.

